docs/conf.py

```diff
@@ -0,0 +1,137 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+
+# pylint: skip-file
+
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'MASCCOT web app'
+copyright = '2019, Tejas Shah'
+author = 'Tejas Shah'
```

Comment:
is it ok if I attach my name here as well?

Reply:
Oh yeah for sure! My bad, but I forgot to change that. You deserve it!